pkg/virt-handler/virtwrap/api/converter.go

```diff
@@ -324,6 +324,43 @@ func Convert_v1_VirtualMachine_To_api_Domain(vm *v1.VirtualMachine, domain *Doma
 		}
 	}
 
+	// Add mandatory console device
```

Comment:
Ideally all of this console and vnc logic should be generic default settings. We can do that at the moment though because the path to the unix file requires knowing the vm's namespace and name a head of time.

This is a result of qemu launching in the libvirt pod instead of the VM pod.  Shared directories are required with unique paths are required for the unix sockets as a work around for the libvirt issue.